### DISCUSSION TITLE
Zoom / Enlarge Scale Playback Feature...

### DISCUSSION THREAD

**Discussion by aphorise** [asciinema/discussions#272] [Ideas]
Offer the ability to zoom or enlarge fonts. Currently typical CTRL++ or CTRL+Scroll enlarges the page with no effect on playback area.

Resize and recrop option of 4:3, 16:9 or other widening or shorterning for example from the top down would also be great (dynamic scrolling).

**Comment by ku1ik** [comment]
The current behaviour is a result of the the site placing the player inside a container (dark grey div) which has its height set to roughly 75% of the viewport height. Zooming in doesn't change the viewport height and therefore it doesn't change terminal text size.

I think the only way to let the terminal zoom as well is to make the container fixed height. This on the other hand wouldn't accomodate (as nicely as it does today) for the vast variety of terminal sizes found in recordings uploaded to asciinema.org

This is a matter of prioritizing one thing over another and I don't have strong opinion but I do slighly prefer current behaviour.
